CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ Changes since v2.4
 - The pdf button and API have been removed. We recommend using "print
   to pdf" in your browser.
 - The `start`, `end` and `expired` fields have been removed from licenses
+- API for creating catalogue item and its localizations has been changed.
+  There is now a single API call that is used to create both a catalogue
+  item and the localizations, namely, /api/catalogue-items/create.
```

Comment:
we seem to be gathering quite a lot of these breaking API changes. we'll soon need to be more careful with them